docs/nx/logger/winston.md

```diff
@@ -0,0 +1,69 @@
+|---------------------|------------------------------------------------------------------|---------|-----------------|---------|-----------|----------------------------------|
+| --name              | Name of the generated library                                    | string  |                 | N/A     | Yes       | Yes                              |
+| --tags              | Add tags to the project (used for linting)                       | string  |                 | N/A     |           |                                  |
+| --directory         | Directory where the project is placed (within Nx libs directory) | string  |                 | N/A     |           |                                  |
+| --skipFormat        | Skip formatting files                                            | boolean | true/false      | false   |           |                                  |
+| --logLevelType      | The type of log levels that will be used                         | enum    | cli/syslog/npm  | npm     |           | Yes                              |
+| --consoleLogs       | Output logs to the console                                       | boolean | true/false      | false   |           |                                  |
+| --fileTransportPath | File path used for logs transport                                | string  |                 | N/A     |           |                                  |
+| --httpTransport     | Add a http transport                                             | boolean | true/false      | false   |           |                                  |
+| --httpTransportHost | Remote host of the HTTP logging endpoint                         | string  |                 | N/A     |           |                                  |
+| --httpTransportPort | Remote port of the HTTP logging endpoint                         | string  |                 | N/A     |           |                                  |
+| --httpTransportPath | Remote URI of the HTTP logging endpoint                          | string  |                 | N/A     |           |                                  |
+| --httpTransportSSL  | Use SSL for the HTTP logging endpoint                            | boolean | true/false      | false   |           |                                  |
+| --streamPath        | Stream transport path                                            | string  |                 | N/A     |           |                                  |
+
+### Generator Output
+##### What is the output of the above commands?
+
+```text title="Files generated"
+.
+├── libs/[libname]
+│   ├── src
+│   ├── ├── index.ts
+│   ├── ├── index.test.ts
+│   ├── .eslintrc.json
+│   ├── jest.config.ts
+│   ├── project.json
+│   ├── tsconfig.json
+│   ├── tsconfig.lib.json
+│   ├── tsconfig.spec.json
+│   ├── README.md
+├── jest.config.ts
+└── jest.preset.ts
+
+```
+
```

Comment:
Could we get a description for these? Maybe comments against each file?

Also, how does output differ depending on the cli options selected, If I select httpTransportOptions for example, what impact does that have on file generation/file contents

Reply:
Have added these.

The files output aren't different depending on the flags, but there are some differences depending on whether this is the first generator run or not - I've included that in the docs (although possibly that belongs in a centralised place).